src/renderer/components/watch-video-info/watch-video-info.js

```diff
@@ -43,10 +47,6 @@ export default defineComponent({
       type: Number,
       required: true
     },
-    subscriptionCountText: {
```

Comment:
Please revert naming changes which is unnecessary to make this work

Reply:
I reversed the naming change. I thought it'd be a good change since in other places it is called subCount, though good point, it is out of scope.

Fixed this in the latest commit